pkg/scheduler/framework/cycle_state.go

```diff
@@ -44,11 +44,11 @@ type StateKey string
 // CycleState does not provide any data protection, as all plugins are assumed to be
 // trusted.
 // Note: CycleState uses a sync.Map to back the storage. It's aimed to optimize for the "write once and read many times" scenarios.
```

Comment:
Can you mention that it's thread safe in Read/Write since we removed the comment below just in case some one not quite familiar with this library.

Reply:
I guess you mean to clarify that sync.Map is safe for concurrently (clarify the type again)? Main thing I removed was  "This function is thread safe by using sync.Map" I guess one way is to do this.. would that work?

```suggestion
// Note: CycleState uses a sync.Map to back the storage, because it is thread safe. It's aimed to optimize for the "write once and read many times" scenarios.
```